app/src/main/java/com/example/immadisairaj/codeforces/SubmissionActivity.java

```diff
@@ -122,6 +133,8 @@ public void onFailure(Call<Submission> call, Throwable t) {
         });
     }
 
+
+
```

Comment:
Can you please remove these two new lines..

Reply:
Is this fixed?

Reply:
@Piyush789 still has not yet responded to this

Reply:
should I?

Reply:
Yeah I am closing this PR. You can work.